docs/volunteering/communication-platforms.md

```diff
@@ -0,0 +1,33 @@
+# Communication Platforms
+
+## Slack
+
+Slack is a realtime messaging platform which allows for synchronous discussion
+in channels and threads.
+
+We use it for discussion, though (due to the nature of the platform) it has
+limitations in this regard.
+
+We occasionally announce things via Slack, however anything important will also
+be sent via email.
+
+## Mailshots (via MailChimp)
+
+We have a general platform for emailing both volunteers and teams in a one-way
+fashion. This is used primarily for important communications where responses can
+happen via another format (for example call for volunteers for an event, with
+responses via a Google Form).
+
+## Mailing lists
+
+We have some Google Groups mailing lists which are used for higher latency
+discussion and for general announcements.
+
```

Comment:
I'd like to add links here to the ones we know we're going to use. @RealOrangeOne I know you were checking them out -- are they ready to go?

Reply:
efd555f15fbeb152f96d33719dcd411847517337 does this.